/**:
  ros__parameters:
    manager:
      odometry: /Odometry
      goal_topic: /goal_pose
      max_vel_: 1.5
      max_acc_: 0.8
      max_jerk_: 1.0
      dynamic_environment: 0
      control_points_distance: 0.5
      cmd_vel: /cmd_vel
      control_cmd_frequency: 100
      safety_cheak_frequency: 20
      plan_cheak_frequency: 20
      control_output: true
      CarType: 0 # 0是圆形底盘 1是矩形底盘
      #矩形底盘长宽,当为圆形底盘时，取Car_L为半径，单位为m
      Car_L : 0.27
      Car_W : 0.40
    sdf_map:
      use_global_map: false
      #没全局地图时使用
      map_size_x: 30.0
      map_size_y: 30.0
      resolusion_: 0.01
      # origin_x: -6.35
      # origin_y: -7.6
      #有全局地图时使用
      global_map_num: 1
      global_map_path:
        - "src/planner_manager/map/RMUC2025.yaml"
        # - "src/planner_manager/map/RMUC1.yaml"

      laser_topic: /scan
      show_esdf_time: false
      local_update_range_x: 3.0
      local_update_range_y: 3.0
      local_map_margin: 10
      frame_id: /odom
      obstacles_inflation: 0.0001
    search:
      max_tau: 0.3
      init_max_tau: 0.4
      lambda_heu: 5.0
      max_vel: 1.5
      max_acc: 0.8
      w_time: 5.0
      horizon: 100.0
      resolution_astar: 0.01
      time_resolution: 0.8
      allocate_num: 100000
      check_num: 30

    optimization:
      max_vel: 1.5
      max_acc: 0.8
      dist0: 0.6
      lambda1: 20.0
      lambda2: 8.0
      lambda3: 0.00001
      lambda4: 0.05
      lambda5: -1.0
      lambda6: -1.0
      lambda7: 100.0
      lambda8: -1.0
      max_iteration_num1: 2
      max_iteration_num2: 300
      max_iteration_num3: 200
      max_iteration_num4: 200
      max_iteration_time1: 0.0001
      max_iteration_time2: 0.005
      max_iteration_time3: 0.003
      max_iteration_time4: 0.003
      algorithm1: 11
      algorithm2: 15
      order: 3
    
    mpc:
      weight_p: 9.0
      weight_yaw: 5.0
      weight_v: 0.1
      weight_w: 0.1


